pillar/heroku/discussions.sls

```diff
@@ -0,0 +1,177 @@
+      'GA_TRACKING_ID': 'UA-5145472-29',
+      'INDEXING_API_USERNAME': 'od_mm_rc_api',
+      'NEW_RELIC_APP_NAME': 'discussions-rc',
+      'MAILGUN_SENDER_DOMAIN': 'discussions-mail.odl.mit.edu',
+      'MICROMASTERS_BASE_URL': 'micromasters-rc.odl.mit.edu',
+      'OCW_LEARNING_COURSE_BUCKET_NAME': 'open-learning-course-data-rc',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsrc',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'odl.mit.edu',
+      'OPEN_DISCUSSIONS_SUPPORT_EMAIL': 'odl-discussions-rc-support@mit.edu',
+      'release_branch': 'release-candidate',
+      'SOCIAL_AUTH_MICROMASTERS_LOGIN_URL': 'https://micromasters-rc.odl.mit.edu/login/edxorg/?next=/discussions/',
+      'vault_env_path': 'rc-apps'
+      },
+    'production': {
+      'app_log_level': 'INFO',
+      'CLOUDFRONT_DIST': 'd2mcnjhkvrfuy2',
+      'DEBUG': False,
+      'env_name': 'production',
+      'ELASTICSEARCH_INDEX': 'discussions',
+      'ELASTICSEARCH_URL': 'https://elasticsearch-production-apps.odl.mit.edu',
+      'GA_TRACKING_ID': 'UA-5145472-30',
+      'INDEXING_API_USERNAME': 'od_mm_prod_api',
+      'NEW_RELIC_APP_NAME': 'discussions-production',
+      'MAILGUN_SENDER_DOMAIN': 'mail.open.mit.edu',
+      'MICROMASTERS_BASE_URL': 'micromasters.mit.edu',
+      'OCW_LEARNING_COURSE_BUCKET_NAME': 'open-learning-course-data',
+      'OPEN_DISCUSSIONS_COOKIE_NAME': 'discussionsprod',
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://open.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'mit.edu',
+      'OPEN_DISCUSSIONS_SUPPORT_EMAIL': 'odl-discussions-support@mit.edu',
+      'release_branch': 'release',
+      'SOCIAL_AUTH_MICROMASTERS_LOGIN_URL': 'https://micromasters.mit.edu/login/edxorg/?next=/discussions/',
+      'vault_env_path': 'production-apps'
+      }
+} %}
+{% set env_data = env_dict[environment] %}
+{% set business_unit = 'mit-open' %}
+{% set pg_creds = salt.vault.cached_read('postgres-{}-apps-opendiscussions/creds/opendiscussions'.format(env_data.env_name), cache_prefix='heroku-opendiscussions') %}
+{% set rds_endpoint = salt.boto_rds.get_endpoint('{env}-rds-postgres-opendiscussions'.format(env=env_data.env_name)) %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  app_name: odl-open-discussions-{{ env_data.env_name }}
+  api_key: __vault__::secret-operations/global/heroku/api_key>data>value
+  config_vars:
+    ALGOLIA_API_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>api_key
+    ALGOLIA_APP_ID: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>app_id
+    ALLOWED_HOSTS: '[*]'
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>secret_key
+    AWS_STORAGE_BUCKET_NAME: 'odl-discussions-{{ env_data.env_name }}'
+    CELERYD_CONCURRENCY: # ONLY IN RC
```

Comment:
If we're not going to use this value we should probably remove it. I don't think it will parse properly if there's nothing assigned to it.